src/js/firstparties/google-search.js

```diff
@@ -1,8 +1,17 @@
 /* globals findInAllFrames:false */
+
+// bool for whether or not in firefox for android
+let firefox_android = (navigator.userAgent.indexOf('Firefox') > -1 && navigator.userAgent.indexOf('Android') > -1);
+
 // In Firefox, outbound google links have the `rwt(...)` mousedown trigger.
 // In Chrome, they just have a `ping` attribute.
 let trap_link = "a[onmousedown^='return rwt(this,'], a[ping]";
```

Comment:
Instead of the new code below, why don't we add another selector to this list, perhaps the same [selector we use in `google-static.js`](https://github.com/EFForg/privacybadger/blob/b275477f272a4edaa60f3699464e4f0a455e708d/src/js/firstparties/google-static.js#L1)?

Reply:
That's how I first tried this, but unfortunately that selector doesn't accurately trap the links in this view. The only reliable way I was able to target and "clean" them was by selecting all links on the page (which led me to put the boolean flag in so that we aren't needlessly targeting all links unless specifically on Firefox Android)

Reply:
OK, looks like we need a different selector. (Why was the selector not matching? What do the search result links look like in the DOM?)

How about selecting on `a[href^='/url?q=']`?